hazelcast/src/main/java/com/hazelcast/multimap/impl/operations/PutOperation.java

```diff
@@ -66,6 +66,7 @@ public void run() throws Exception {
     @Override
     public void afterRun() throws Exception {
         if (Boolean.TRUE.equals(response)) {
+            getOrCreateContainer().update();
```

Comment:
This is probably the most important fix regarding the `lastUpdateTime` -> it wasn't modified on put operations